src/ngraph/pass/opset0_downgrade.cpp

```diff
@@ -146,13 +146,48 @@ namespace
     bool op_cast(shared_ptr<op::v1::Broadcast> node)
```

Comment:
Do we need to update upgrade pass as well?

Reply:
I've refactored it.